docs/Meadow/Getting_Started/Getting_Started_Core-Compute_Module/index.md

```diff
@@ -0,0 +1,104 @@
+
+1. **Install Visual Studio** - Download and Install [Visual Studio](https://visualstudio.microsoft.com/) for either Windows or macOS to prepare your development machine. Community edition will work fine.
+1. **Deploy Meadow.OS** - Use the Meadow.CLI to [deploy Meadow.OS to your board](/Meadow/Getting_Started/Deploying_Meadow/), booting the Core-Compute module into bootloader mode when you first connect it to your development machine.
+1. **Deploy Hello, World** - Verify everything is setup by deploying your first Meadow application. You can start working from the Feather [Hello, Meadow](/Meadow/Getting_Started/Hello_World/) application, but you will need to connect external components or peripherals to recreate the experience of the onboard RGB LED. You can still see logging output without any additional components.
+
+```csharp
+using Meadow;
+using Meadow.Devices;
+using Meadow.Foundation.Leds;
+using System;
+using System.Threading.Tasks;
+
+namespace HelloMeadow
+{
+    // Change F7FeatherV2 or F7FeatherV1 for Feather boards
+    public class MeadowApp : App<F7CoreComputeV2>
+    {
+        public override Task Run()
+        {
+            Console.WriteLine("Run...");
+
+            return base.Run();
+        }
+
+        public async override Task Initialize()
+        {
+            Console.WriteLine("Initialize...");
+
+            await base.Initialize();
+        }
+    }
+}
+```
+
+You should see the console output generated from the Meadow showing up in the Meadow output pane within Visual Studio.
+
+For an example with some visual flare, you could connect an LED's anode leg to the Meadow ground (GND) pin and its cathode leg to pin **D14**. Then, within your app's `Run` method, you can start blinking it using the `Led` class in [Meadow.Foundation](..\Meadow.Foundation\Getting_Started\).
+
+```csharp
+using Meadow;
+using Meadow.Devices;
+using Meadow.Foundation.Leds;
+using System;
+using System.Threading.Tasks;
+
+namespace HelloMeadow
+{
+    // Change F7FeatherV2 or F7FeatherV1 for Feather boards
+    public class MeadowApp : App<F7CoreComputeV2>
+    {
+        Led led;
+
+        public override Task Run()
+        {
+            Console.WriteLine("Run...");
+
+            led.StartBlink(TimeSpan.FromSeconds(1), TimeSpan.FromSeconds(0.5));
+
+            return base.Run();
+        }
+
+        public async override Task Initialize()
+        {
+            Console.WriteLine("Initialize...");
+
+            led = new Led(Device.CreateDigitalOutputPort(Device.Pins.D14));
+
+            await base.Initialize();
+        }
+    }
+}
+```
+
+## Core-Compute pin details
+
+When using the Core-Compute module within the Core-Compute Development Kit, some pins available via the Feather pin breakout headers have extra considerations. are reserved for use by other components on the development kit.
```

Comment:
Looks like it's missing a word or two on the last sentence

Reply:
Oops...it definitely seems so. Let me fix that.

Reply:
I think it was an old revision of that intro sentence. It's gone now.